jetty-server/src/main/java/org/eclipse/jetty/server/handler/InetAccessHandler.java

```diff
@@ -188,13 +188,16 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
     protected boolean isAllowed(InetAddress addr, Request baseRequest, HttpServletRequest request)
     {
         String name = baseRequest.getHttpChannel().getConnector().getName();
+        Boolean filterAppliesToConnector = _names.test(name);
```

Comment:
Why is one a Boolean and the other a boolean?
Also, can the Boolean ever be null, in which case line 198 can throw NPE.